src/PrMerger.js

```diff
@@ -109,7 +129,11 @@ class PrMerger {
         }
         const prNum = Util.ParseTag(tag.ref);
         Logger.info("PR" + prNum + " is the current");
+        // TODO: handle 'not found' errors
         const stagingPr = await GH.getPR(prNum, false);
+        assert(stagingPr);
+        stagingPr.clearedForMerge = await this._clearedForMerge(prNum);
```

Comment:
Sooner or later, we will forget to initialize this member. Let's initialize it in the PR constructor and only there. AFAICT, such initialization is only needed in the guarded run mode.